src/cascade/executor/cascade_plan.py

```diff
@@ -82,19 +88,50 @@ def cascade_job(self, cascade_job_id):
             grandparent_location_id = None
 
         print(f"settings {type(self._settings)} {self._settings.policies}")
+        parent_location_id = self._location_of_cascade_job(cascade_job_id)
+
+        policies = policies_from_settings(self._settings)
         local_settings = EstimationParameters(
             settings=self._settings,
-            policies=policies_from_settings(self._settings),
-            locations=self._locations,
-            parent_location_id=self._location_of_cascade_job(cascade_job_id),
-            grandparent_location_id=grandparent_location_id
+            policies=policies,
+            children=list(sorted(self._locations.successors(parent_location_id))),
+            parent_location_id=parent_location_id,
+            grandparent_location_id=grandparent_location_id,
+            sex_id=self._settings.model.drill_sex,
         )
+        local_settings.data_access = _ParameterHierarchy(**dict(
+            gbd_round_id=self._settings.gbd_round_id,
+            modelable_entity_id=self._settings.model.modelable_entity_id,
+            model_version_id=self._settings.model.model_version_id,
+            settings_file=self._args.settings_file,
+            bundle_file=self._args.bundle_file,
+            bundle_id=self._settings.model.bundle_id,
+            bundle_study_covariates_file=self._args.bundle_study_covariates_file,
+            tier=2 if self._args.skip_cache else 3,
```

Comment:
Good, it's nice to have this tucked into a place where it's out of the way but clear